tsdb/head.go

```diff
@@ -343,6 +347,14 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 			Name: "prometheus_tsdb_mmap_chunk_corruptions_total",
 			Help: "Total number of memory-mapped chunk corruptions.",
 		}),
+		sparseHistogramsAddedTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "prometheus_tsdb_sparse_histogram_added_total",
+			Help: "Total number of sparse histograms added.",
+		}),
+		sparseHistogramSeries: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "prometheus_tsdb_sparse_histogram_series",
```

Comment:
Using "series" here sounds like it's different from "histograms". But IIUC, it's the same. Ideally, this metric would just be called `prometheus_tsdb_sparse_histograms` (and the counter above `prometheus_tsdb_sparse_histograms_total` as suggested). Sadly, OpenMetrics decided to retroactively make this invalid in expositions).

How about `prometheus_tsdb_sparse_histograms_in_memory` or `prometheus_tsdb_sparse_histograms_in_head` (if that's more precise)?

Reply:
So the other metrics counts total histograms added, while this is number of series having histograms.

If we have like 
```
s1 -> [ (t1, h1), (t2, h2) ]
s2 -> [ (t1, h3) ]
```

This metric would be 2, while the other (_total) metric would be 3. So wouldn't having `series` in the name make sense here?